### DISCUSSION TITLE
Fragement d'arme

### DISCUSSION THREAD

**Discussion by Archimageifrit** [Crownicles/Crownicles#1107] [Nouvelles idées]
Bon du coup j'ai déjà proposé l'idée mais comme j'avais pas mis plusieurs idée on recommence .
Donc l'idée serait que à la place de juste vendre nos items on a aussi le choix de pouvoir les "brisé" pour récupéré des fragments , fragments qui pourrons être utilisé de divers moyen : 
- Upgrade d'équipement ( soit stats ou vraiment upgrades , comme par exemple transformer un livre d'incantation Il en livre d'incantation III ) 
-  Un système de forge ou on pourrai utiliser des fragments pour craft des items

**Comment by ghost** [comment]
j'aime l'idée mais j'ai peur que ca soit trop complexe pour un bot discord

**Reply by Archimageifrit** [reply]
Bah enfaite je pense pas , ça serait une sorte de nouvelles money comme les gemme ou l'argent , après pour l'ouvrage et tout aucune idée je sais pas comment fonctionne le bot

**Reply by ghost** [reply]
mais rajouter un marchand de type forgeron (même genre que le shop de mission) ou on peut demonter son arme ou acheter une arme/bouclier avec des fragments ca peut etre cool
marchand trouvable que dans les grandes villes bien sur
